api/controllers/MapController.js

```diff
@@ -48,24 +165,251 @@ module.exports = {
                 "lat":54.2,"lng":42.8
             }
         }
-		fs.readFile('cities.json', "utf-8",function(err,data){
-			var cities = JSON.parse(data)
-			var cities = cities.features;
+		fs.readFile('GeoJson/cities.geojson', "utf-8",function(err,data){
+			var data = JSON.parse(data);
+			var cities = data.features
 			var cityNames = [];
+			var count = 0;
 			for (var i = cities.length - 1; i >= 0; i--) {
+				if(cities[i].properties.population == undefined){continue}
 				var lat = cities[i].geometry.coordinates[1];
 				var lng = cities[i].geometry.coordinates[0];
 				if(lat >= bounds._southWest.lat && lat <= bounds._northEast.lat){
 					if(lng >= bounds._southWest.lng && lng <= bounds._northEast.lng){
-					cityNames.push(cities[i]);
+						var city = {};
+						
+						var roundedLat = Math.round(((2*Math.round(cities[i].geometry.coordinates[0]*50)/100)+0.01)*100)/100;
+
+						var roundedLng = Math.round(((25*Math.round(cities[i].geometry.coordinates[1]*40)/1000)+0.0125)*1000)/1000;
+						city.geometry = {"type":"Point","coordinates":[roundedLng,roundedLat]}
+
+						city.type = "Feature";
+						city.id = cities[i].id;
+						city.properties = {};
+						city.properties.name = cities[i].properties.name;
+						if (cities[i].properties["name:en"]){
+							city.properties.name = cities[i].properties["name:en"]
+						}
+						city.properties.is_in = cities[i].properties.is_in;
+						if (cities[i].properties["is_in:country"]){
+							city.properties.is_in = cities[i].properties["is_in:country"]
+						}
+						if(cities[i].properties.ele){city.properties.ele = cities[i].properties.ele};
+						var isIn = "";
+						if(city.properties.is_in){isIn = city.properties.is_in;}
+						if(isIn.includes("Czech")){
```

Comment:
you could break all of these into a json or yml file  then loop through that file.  this is fine for now., but add a @TODO